source/Sashimi.Tests.Shared/Server/ActionHandlerTestBuilder.cs

```diff
@@ -25,23 +40,69 @@ public static ActionHandlerTestBuilder<TCalamari> Create<TCalamari>(Type actionH
             return new ActionHandlerTestBuilder<TCalamari>(actionHandlerType);
         }
 
-        public static TestActionHandlerContext<TCalamariProgram> WithPackage<TCalamariProgram>(this TestActionHandlerContext<TCalamariProgram> context, string path)
-            where TCalamariProgram : CalamariFlavourProgram
+        public static TestActionHandlerContext<TCalamariProgram> WithFilesToCopy<TCalamariProgram>(this TestActionHandlerContext<TCalamariProgram> context, string path)
```

Comment:
What's the use case for `WithFilesToCopy` vs. `WithPackage`?

Reply:
WithFilesToCopy suppresses the need to package files for testing